config/development.js

```diff
@@ -1,7 +1,8 @@
+require('dotenv').config();
 module.exports = {
   contentstack: {
-    api_key: 'Please Enter your Stack api_key',
-    access_token: 'Please Enter your Stack access_token',
-    environment: 'Environment'
+    api_key: process.env.DB_USER,
+    access_token: process.env.DB_PASS,
```

Comment:
Could you please also change 
`access_token: process.env.ACCESS_TOKEN` to `delivery_token: process.env.DELIVERY_TOKEN`.

There is a gap in `Getting Started` guide. I have already notified the course creators.

Reply:
Don't forget to apply the change to `server.js`